src/util/notifications.ts

```diff
@@ -157,6 +158,46 @@ async function showNotification(
 	return notificationId;
 }
 
+/**
+ * Show notification warning about native scrobbler
+ * @param connector - Connector for which to show it
+ */
+export async function showNativeScrobblerWarning(
+	connector: ConnectorMeta,
+): Promise<void> {
+	const nativeScrobblerNotification = new NativeScrobblerNotification();
+	if (
+		!connector.hasNativeScrobbler ||
+		!(await nativeScrobblerNotification.shouldNotifyAboutNativeScrobbler(
+			connector.id,
+		))
+	) {
+		return;
+	}
+
+	const options = {
+		title: browser.i18n.getMessage(
+			'notificationNativeScrobbler',
+			connector.label,
+		),
+		message: browser.i18n.getMessage('notificationNativeScrobblerText'),
+	};
+
+	try {
+		await showNotification(options, () => {
+			browser.tabs.create({
+				url: browser.runtime.getURL(
+					'src/ui/options/index.html?p=connectors',
+				),
+			});
+		});
+		await nativeScrobblerNotification.saveHasNotified(connector.id);
```

Comment:
:bulb: what about making the notification such that it has to be actively dismissed to not come up again.

I can imagine folks would miss this? I guess for now this is a good first step and we can always iterate though.

Reply:
Yeah it is a little unlucky, but unfortunately I'm not aware of any nice UX way to do this with notifications (no custom buttons on notifs).

If so we'd have to in that case have that be on the user manually closing the notif before it disappears on its own using the normal "X" button, but that seems very unintuitive to me, and I think a fair number of people will never click to close it and be annoyed that it keeps popping up all the time.

Alternatively we'd have to go into dom injection which seems annoying to maintain.

Reply:
Lets go with this and we can always iterate if folks still have problems :)